common/djangoapps/student/views.py

```diff
@@ -1740,6 +1747,10 @@ def create_account_with_params(request, params):
     if third_party_auth.is_enabled() and pipeline.running(request):
         running_pipeline = pipeline.get(request)
         third_party_provider = provider.Registry.get_from_pipeline(running_pipeline)
+        # EnterpriseCustomers (part of the `enterprise` app) tied to a third-party
```

Comment:
So, it'll always get added to the pipeline kwargs, but it'll only get used if the Enterprise pipeline elements are inserted (that is, if the Enterprise app is enabled). The pipeline kwargs are pretty flexible, so if we stick consent in there (which will be None if Enterprise isn't enabled), it just won't get used.